aws_cloudformation_policies/aws_cloudformation_stack_drifted.yml

```diff
@@ -79,7 +79,7 @@ Tests:
   -
     Name: Stack Drifted but Ignored
     ResourceType: AWS.CloudFormation.Stack
-    ExpectedResult: false
+    ExpectedResult: true
```

Comment:
true == compliant, right? In that case, yes this should be true. I didn't run tests locally in #98 because I assumed they would run in CI, my bad. I still get confused on true/false for policy vs rule sometimes, good catch

Double-check the rule logic if you haven't already (make sure it isn't also flipped in one of the branches)

Reply:
That's correct, true == compliant for policies. I think CI actually  just isn't running in this repo right now which isn't good. I checked the policy and it looks good to me.